src/MembershipManager.php

```diff
@@ -305,9 +305,9 @@ public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $gro
       }
 
       // Compile a list of group target IDs.
-      $target_ids = array_map(function ($value) {
-        return $value['target_id'];
-      }, $entity->get($field->getName())->getValue());
+      $target_ids = array_filter(array_map(function ($value) {
+        return isset($value['target_id']) ? $value['target_id'] : NULL;
```

Comment:
See https://www.php.net/manual/en/migration70.new-features.php
```suggestion
        return $value['target_id'] ?? NULL;
```